workspaces/orchestrator/plugins/orchestrator-backend/src/service/OrchestratorService.ts

```diff
@@ -221,12 +217,18 @@ export class OrchestratorService {
     cacheHandler?: CacheHandler;
   }): Promise<WorkflowOverview | undefined> {
     const { definitionId, cacheHandler } = args;
-    const isWorkflowAvailable = this.workflowCacheService.isAvailable(
-      definitionId,
-      cacheHandler,
-    );
-    return isWorkflowAvailable
-      ? await this.sonataFlowService.fetchWorkflowOverview(definitionId)
-      : undefined;
+    let isWorkflowAvailable;
+    try {
+      isWorkflowAvailable = this.workflowCacheService.isAvailable(
+        definitionId,
+        cacheHandler,
+      );
+    } catch (e) {
```

Comment:
```suggestion
    } catch {
```

Reply:
I suggest to not pass cachehandler and to remove try catch.